apps/web/src/components/ai/DataAgentChatAccessibility.tsx

```
@@ -0,0 +1,618 @@
+            key={index} 
+            className="p-3 border rounded-md focus-within:ring-2 focus-within:ring-[#AC9469] focus-within:ring-offset-2"
+            role="article"
+            aria-labelledby={`${itemId}-client`}
+            aria-describedby={`${itemId}-details`}
+          >
+            <div id={`${itemId}-client`} className="font-medium text-sm">{item.clientName}</div>
+            <div id={`${itemId}-details`} className="text-xs text-muted-foreground">
+              <span>{item.serviceName}</span>
+              <span className="mx-2">•</span>
+              <span aria-label={`Valor: ${formatCurrency(item.amount)}`}>
+                {formatCurrency(item.amount)}
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)